### DISCUSSION TITLE
账号登录问题

### DISCUSSION THREAD

**Discussion by p426575** [natfrp/wiki#209] [水深火热] [answered]
以前的账号在系统重装后突然不见了，只有以前的右键，咋办《再注册个账号，能二次实名吗

**Comment by fakeboboliu** [comment]
你好，请使用找回密码。

**Reply by aylitat** [reply]
我以前好多年的账号似乎被清空了？剩余几百G流量都是辛辛苦苦一天天签到攒出来的。。。

**Comment by p426575** [comment]
不是，我连账户是什么也不知道，只有之前那个邮件，登录那个右键又是我新创的号，但我实名只能一个账号，没法用




------------------&nbsp;Original message&nbsp;------------------
From: "bobo liu"; 
Sendtime: Saturday, Nov 12, 2022 8:32 PM
To: "natfrp/wiki"; 
Cc: ***@***.***>; "Author"; 
Subject: Re: [natfrp/wiki] 账号登录问题 (Discussion #209)





 
你好，请使用找回密码。

**Reply by fakeboboliu** [reply]
1. 请使用回复功能
2. 请理解下自己在说什么，这里是 GitHub 不是支持渠道，我们希望在这里看到有逻辑的深度用户

**Reply by fakeboboliu** [reply]
另：忘记密码可以使用邮箱

**Comment by p426575** [comment]
就是说我之前的账号连账号是什么都忘了，只有注册邮件，找回密码显示不存在，我说的是樱花映射




------------------&nbsp;Original message&nbsp;------------------
From: "bobo liu"; 
Sendtime: Saturday, Nov 12, 2022 8:39 PM
To: "natfrp/wiki"; 
Cc: ***@***.***>; "Author"; 
Subject: Re: [natfrp/wiki] 账号登录问题 (Discussion #209)





  
请使用回复功能
 
请理解下自己在说什么，这里是 GitHub 不是支持渠道，我们希望在这里看到有逻辑的深度用户

**Reply by fakeboboliu** [reply]
您好，
这是您第二次为此社区增添混乱，我将在下次将您拉黑

**Comment by p426575** [comment]
我那个号没有绑定，但是却有邮件，我也不知道为啥，就是很奇怪




------------------&nbsp;Original message&nbsp;------------------
From: "bobo liu"; 
Sendtime: Saturday, Nov 12, 2022 8:41 PM
To: "natfrp/wiki"; 
Cc: ***@***.***>; "Author"; 
Subject: Re: [natfrp/wiki] 账号登录问题 (Discussion #209)





 
另：忘记密码可以使用邮箱

**Reply by fakeboboliu** [reply]
本社区并不是我们的支持渠道，
为了防止给此社区的围观者添麻烦，您将因为多次违背我们认为一名 GitHub 用户应在此平台的表现而被在 GitHub Discussions 拉黑。

如果需要支持请在确认有问题后使用支持渠道（即邮件）。

**Comment by p426575** [comment]
?????我怎么了




------------------&nbsp;Original message&nbsp;------------------
From: "bobo liu"; 
Sendtime: Saturday, Nov 12, 2022 8:42 PM
To: "natfrp/wiki"; 
Cc: ***@***.***>; "Author"; 
Subject: Re: [natfrp/wiki] 账号登录问题 (Discussion #209)





 
您好，
 这是您第二次为此社区增添混乱，我将在下次将您拉黑

**Comment by fakeboboliu** [comment, marked as answer]
本社区并不是我们的支持渠道，
为了防止给此社区的围观者添麻烦，您将因为多次违背我们认为一名 GitHub 用户应在此平台的表现而被在 GitHub Discussions 拉黑。

如果需要支持请在确认有问题后使用支持渠道（即邮件 support@natfrp.cloud ）。
